src/utils/__tests__/url-test.js

```diff
@@ -12,15 +12,15 @@ describe('getFullUrl', () => {
   });
 });
 
-describe('getResourceWithAuth', () => {
+describe('getResource', () => {
   test('when uri contains domain, do not change, add auth headers', () => {
     const expectedResult = {
       uri: 'https://example.com/img.gif',
       headers: {
         Authorization: 'Basic am9obmRvZUBleGFtcGxlLmNvbTpzb21lQXBpS2V5', // eslint-disable-line
```

Comment:
Might be clearer to use the `getAuthHeader` function here instead of hardcoding the base 64